src/views/tutorial-view/components/feedback-panel/index.tsx

```diff
@@ -0,0 +1,108 @@
+import { IconThumbsDown16 } from '@hashicorp/flight-icons/svg-react/thumbs-down-16'
+import { IconThumbsUp16 } from '@hashicorp/flight-icons/svg-react/thumbs-up-16'
+import type { FeedbackFormProps } from 'components/feedback-form/types'
+import FeedbackForm from 'components/feedback-form'
+import s from './feedback-panel.module.css'
+
+/**
+ * Largely copied from: https://github.com/hashicorp/learn/blob/master/components/feedback-panel/index.jsx
+ */
+
+async function recordFeedback(responses, sessionId) {
+	const body = {
+		responses: responses.reduce(
+			(obj, { id, value }) =>
+				Object.assign(obj, {
+					[id]: value,
+				}),
+			{}
+		),
+		sessionId,
+		timestamp: new Date(),
+	}
+
+	try {
+		const payload = {
+			...body,
+			page: document.location.pathname,
+		}
+
+		const apiRequest = async () => {
+			const res = await fetch('/api/feedback', {
```

Comment:
We don't have an `api/feedback` endpoint yet on dev dot right? We should probably create a new form for this data, or I suppose we could use the old tutorials form 🤔